web/src/lib/components/asset-viewer/editor/crop-canvas.svelte

```diff
@@ -0,0 +1,740 @@
+<script lang="ts">
+  import {
+    cropAspectRatio,
+    cropImageScale,
+    cropImageSize,
+    cropSettings,
+    type CropSettings,
+    type CropAspectRatio,
+    cropSettingsChanged,
+  } from '$lib/stores/asset-editor.store';
+  import { onMount, afterUpdate } from 'svelte';
+  import { get } from 'svelte/store';
+
+  export let crop = get(cropSettings);
```

Comment:
Use the dollar sign syntax for stores inside components
- `$store` to read the value instead of get() or .subscribe()
- `$store = newValue` to update the value instead of .set()

Reply:
Thanks, I'll remember. I just started getting acquainted with svelte in this project, before that I worked with other frameworks